models/luismodel.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Can we just re-use the utterances model in this folder, or is there some reason we need an additional model? I know we have a gap in test coverage to train from a model...

Reply:
done